docs/deploy/docker.md

```diff
@@ -42,3 +43,5 @@ Note that you’ll have to set a [`SECRET_KEY`](https://docs.djangoproject.com/e
 ```sh
 $ python3 -c 'import secrets; print(secrets.token_hex(100))'
 ```
+
+User accounts can be managed via the Django admin page ({SITE_URL}/admin). The required superuser must be created on the container command line with this command: umap createsuperuser.
```

Comment:
For commands, it's better to put them on a new line, like this:

````markdown
```bash
umap createsuperuser
```
````